### ISSUE TITLE
Terms of use for my.cdash.org?

### ISSUE THREAD

**Issue by chhtz** [Kitware/CDash#909]
I could not find any kind of "Terms of use" on that site. 
I assume it is essentially "free to use" and "provided as is"?

**Comment by zackgalbreath** [comment]
Yes, that's right. You are free to create a publicly visible project there. We provide you with up to 500 builds for your project. By default old builds expire after 60 days.

**Comment by chhtz** [comment]
Thanks for the answer!

Actually, I have the same question for `open.cdash.org` or more importantly, where to apply for a project at that sub-site? (I'm asking on behalf of [Eigen](http://eigen.tuxfamily.org) -- but we are fine with my.cdash.org as well ...)

And I really recommend adding some basic terms of use information to the website.
